### PR TITLE
docs(cx-elements, cx-layout): Update Token Shadowing docs

### PR DESCRIPTION
**Pull request by misterjones**
## Changes

* Split Token Shadowing out of "CX Component Template" page into separate page.
* Perform standard doc cleanup (typos, appease the linter, 100 col width, etc.).
* Update shadow paths in CX Layout pages.

### Documents:

* #1466

### CODE REVIEW

File: packages/cx-elements/doc/ComponentTemplate.md [hunk 1]
```diff
@@ -150,295 +151,166 @@ If you have both static and dynamic versions of your token collection:
...
-
-Both supported use-cases are implemented by following the following steps:
-- Add the bodiless static file replacement plugin to your webpack configuration
-  using the `addStaticReplacementPlugin` function from `@bodiless/webpack`:
-  ```
-  const newConfig = addStaticReplacementPlugin(existingConfig);
-  ```
-  This should be added only when building production bundles, and *must not* be
-  added for server-side rendering.
+## Static Replacement
```
Comment (hvanyo) [inline packages/cx-elements/doc/ComponentTemplate.md:225]: @misterjones  @wodenx  I feel this deserves its own page. I don't know if I would look under component templates.

File: packages/cx-elements/bodiless.docs.json [hunk 2]
```diff
@@ -5,8 +5,9 @@
       "README.md": "./doc/Components.md",
       "CX_Elements": {
         "README.md": "./doc/CX_Elements.md",
-        "CX_SiteTypography.md": "./doc/CX_SiteTypography.md",
-        "CX_Shadow.md": "./doc/CX_Shadow.md"
+        "CX_ComponentTemplate.md": "./doc/ComponentTemplate.md",
```
Comment (hvanyo) [inline packages/cx-elements/bodiless.docs.json:8]: @misterjones  I think this should be ComponentTemplate

File: packages/cx-elements/bodiless.docs.json [hunk 3]
```diff
@@ -5,8 +5,9 @@
...
+        "CX_Shadow.md": "./doc/CX_Shadow.md",
```
Comment (hvanyo) [inline packages/cx-elements/bodiless.docs.json:9]: @misterjones I think this should be Shadowing without CX prefix

Comment (misterjones) [reply packages/cx-elements/bodiless.docs.json]: ✅  Removed CX prefix.

File: packages/cx-elements/bodiless.docs.json (hunk 2 above)
Comment (misterjones) [reply packages/cx-elements/bodiless.docs.json]: ✅  Removed CX prefix.

File: packages/cx-elements/doc/ComponentTemplate.md (hunk 1 above)
Comment (misterjones) [reply packages/cx-elements/doc/ComponentTemplate.md]: @hvanyo Whoops — as I had moved the shadow documentation out into its own page, I had meant to ask about moving this section out as well.

✅  Moved "Static Replacement" out into its own page (see: [Static Replacement](https://github.com/johnsonandjohnson/Bodiless-JS/pull/1484/files#diff-35ff16ebbb943f4d3cd99f94b5e11cf9e6c6c07e48957997d6595cebee962af9)).

**Review by hvanyo — APPROVED** [review]

**Review by wodenx — APPROVED** [review]

**Pull request merged**